### DISCUSSION TITLE
No data connection from driver station to gizmo

### DISCUSSION THREAD

**Discussion by NG260084** [gizmo-platform/gizmo#31] [Q&A]
Whenever they gizmo boots up, the main LEDs flash red for network, blank for field, and green for battery. I've read that this means that there is no data between the driver station and the gizmo itself. I tried to connect them manually using the tether cable, however the lights won't change. The driver is giving the green and orange lights on its ethernet port so it is powered, and all the other power lights on the gizmo are on. What should I do to get the two stations connected?

**Comment by the-maldridge** [comment]
This sounds like you are not in a paired state.  Connect the USB programming cable between the system processor and any USB port on the driver station, boot both and wait for them to come up (~1 minute), then push and hold the BOOTSEL button on the system processor.  The field and network lights will briefly flash red and the system processor will reboot.  After this point you should have both devices paired, and the pairing will be retained.
